### ISSUE TITLE
Show the dropbox uri when syncing to solve http 400 errors

### ISSUE THREAD

**Issue by ericchartier** [Laverna/laverna#423]
When trying to sync a new instance of laverna with dropbox, I have to add the uri key to the app config on dropbox, but I have no idea what that uri is.

What I'm using on the website laverna.cc no longer works with the standalone installation on Windows. On the website it is either https://laverna.cc/app/dropbox.html or https://laverna.cc/dropbox.html, not sure which one is good anymore.

In the past, dropbox error messages would show the invalid uri but in that precise case it does not appear.

**Comment by ericchartier** [comment]
How am I supposed to sync with dropbox on standalone version?

**Comment by CrystalMethod** [comment]
I'm also interested in this use case. :+1:

**Comment by colinleefish** [comment]
I encountered exactly the same problem. Any help? Thanks very much!

**Comment by wwebfor** [comment]
All of you are using your own API key for Dropbox, correct?

**Comment by CrystalMethod** [comment]
@colinleefish Thanks for the hint to the correct redirect_uri.

Using 0.7.2-RC3 I was able to sync Laverna desktop with my Dropbox account.

**Comment by colinleefish** [comment]
@wwebfor I'm using the API key I created as instructed in the config page.

**Comment by jn0** [comment]
![laverna-dropbox](https://cloud.githubusercontent.com/assets/407437/14042518/0bfc6b78-f294-11e5-8179-be43ae08c9aa.png)
MacOS 10.10.5, Laverna [from here](https://codeload.github.com/Laverna/static-laverna/zip/gh-pages).
Attempt to setup the sync to dropbox failed.

**Comment by CrystalMethod** [comment]
No, you're using the static version supposed to be used for self hosting Laverna. Please try the native application to be found here: https://github.com/Laverna/laverna/releases/tag/0.7.2-RC3 .

**Comment by varod** [comment]
in 0.7.2-RC4 laverna listens on TCP/9000 instead of TCP/9100 which means we have to add `http://localhost:9000/dropbox.html`in the redirect uri's Dropbox application settings

**Comment by winuxue** [comment]
I have the same issue in linux
